format/src/main/thrift/carbondata.thrift

```diff
@@ -1,223 +1,226 @@
+ * For V1 format.
+ */
 struct DataChunk{
-    1: required ChunkCompressionMeta chunk_meta; // the metadata of a chunk
-    2: required bool rowMajor; // whether this chunk is a row chunk or column chunk ? Decide whether this can be replace with counting od columnIDs
+    1: required ChunkCompressionMeta chunk_meta; // The metadata of a chunk
+    2: required bool rowMajor; // Whether this chunk is a row chunk or column chunk, Decide whether this can be replace with counting of columnIDs.
 	/** The column IDs in this chunk, in the order in which the data is physically stored, will have atleast one column ID for columnar format, many column ID for row major format**/
     3: required list<i32> column_ids;
     4: required i64 data_page_offset; // Offset of data page
-    5: required i32 data_page_length; // length of data page
-    6: optional i64 rowid_page_offset; //offset of row id page, only if encoded using inverted index
-    7: optional i32 rowid_page_length; //length of row id page, only if encoded using inverted index
-    8: optional i64 rle_page_offset;	// offset of rle page, only if RLE coded.
-    9: optional i32 rle_page_length;	// length of rle page, only if RLE coded.
-    10: optional PresenceMeta presence; // information about presence of values in each row of this column chunk
+    5: required i32 data_page_length; // Length of data page
+    6: optional i64 rowid_page_offset; // Offset of row id page, only if encoded using inverted index
+    7: optional i32 rowid_page_length; // Length of row id page, only if encoded using inverted index
+    8: optional i64 rle_page_offset;	// Offset of rle page, only if RLE coded.
+    9: optional i32 rle_page_length;	// Length of rle page, only if RLE coded.
+    10: optional PresenceMeta presence; // Information about presence of values in each row of this column chunk
     11: optional SortState sort_state;
     12: optional list<schema.Encoding> encoders; // The List of encoders overriden at node level
-    13: optional list<binary> encoder_meta; // extra information required by encoders
+    13: optional list<binary> encoder_meta; // Extra information required by encoders
 }
 
 /**
-* Represents a chunk of data. The chunk can be a single column stored in Column Major format or a group of columns stored in Row Major Format.
-**/
+ * Represents a chunk of data. The chunk can be a single column stored in Column Major format or a group of columns stored in Row Major Format.
+ * For V2 format.
+ */
 struct DataChunk2{
-    1: required ChunkCompressionMeta chunk_meta; // the metadata of a chunk
-    2: required bool rowMajor; // whether this chunk is a row chunk or column chunk ? Decide whether this can be replace with counting od columnIDs
+    1: required ChunkCompressionMeta chunk_meta; // The metadata of a chunk
+    2: required bool rowMajor; // Whether this chunk is a row chunk or column chunk, Decide whether this can be replace with counting of columnIDs.
```

Comment:
`Decide whether this can be replace with counting of columnIDs` we can remove this part as we are deciding based on the schema file.